app/services/feeds/assemble_article_markdown.rb

```diff
@@ -31,6 +31,13 @@ def call
 
     private
 
+    def base_url
+      @base_url ||=
+        URI.parse(@item.url).then do |uri|
```

Comment:
are there ever rss items without a url?

https://web.resource.org/rss/1.0/spec#s5.5 the RSS spec requires there be a valid uri for each item
https://validator.w3.org/feed/docs/atom.html#requiredEntryElements the atom spec uses id and requires it be a unique URI.